lib/ts/recipe/emailverification/emailVerificationAuth.tsx

```diff
@@ -33,15 +34,15 @@ const EmailVerificationAuth: React.FC<Props> = ({ children, ...props }) => {
     const doesSessionExist = sessionContext.doesSessionExist;
     const emailVerificationMode = props.recipe.config.mode;
     const propsRef = React.useRef(props);
+    const userContext = useUserContext();
```

Comment:
When this is used inside emailpasswordauth, it passes the context via props. So then that will not be considered here.

Reply:
For EmailVerificationAuth and SessionAuth im not sure how to handle both cases
- being used in other recipe components
- being used directly as a component by the user

In the SDK currently all uses of the two are wrapped by `UserContextWrapper` but if the user adds these manually there is no way to pass userContext as props in a clean way. We should discuss this

Reply:
We discussed that it can be done in the same way as everything else.